docs/cn/instructions/quickstart.zh-CN.md

```diff
@@ -6,6 +6,7 @@
 建议使用64位操作系统，建议使用Linux / Unix；
 64位JDK 1.8+;
 Gradle至少为5.6, 推荐 5.6.*
+idea，推荐使用2021版本
```

Comment:
Idea is not a dependency.
The gradle 5.6 may not enough, it seems we should use gradle version 7.0? And we recommend to use gradlew to build the project.

Reply:
OK.